app/poros/neo_search.rb

```diff
@@ -8,33 +8,42 @@ def birthday_data(birthdate)
     NeoService.new.birthday(birthdate)
   end
 
+  def daily_message
+    unless DailyMessage.where(date: Date.today.to_s).empty?
+      return @message = DailyMessage.where(date: Date.today.to_s).first
+    else
+      daily_neos
+    end
+  end
```

Comment:
this checks to see if there is already a daily message and uses it if there is one. if not, then it generates one